docs/guides/using-org-slugs.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: 'Using organization slugs in URLs'
+description: Learn how to use organization slugs in URLs to manage the active Clerk organization.
+---
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Next.js + Clerk application",
+      link: "/docs/quickstarts/nextjs",
+      icon: "nextjs",
+    },
+    {
+      title: "Enable organizations for your instance",
+      link: "/docs/organizations/overview",
+      icon: "globe",
+    },
+    {
+      title: "Explore the full sample repository on GitHub",
+      link: "https://github.com/clerk/orgs/tree/main/examples/sync-org-with-url",
+      icon: "git",
+    }
+  ]}
+>
+  - Allow users to choose an organization slug
+  - Include an organization slug in your URLs
+  - Render organization-specific content
+</TutorialHero>
+
+## Introduction
+
+In this guide, you will learn how to include an organization slug in your application's URLs, and
+how to render information about the user's active organization at runtime.
+
+## Background
+
+With [Organizations](/docs/organizations/overview), users can create entities that represent their companies within your
+application. Users can be members of multiple organizations, but can only have one
+[active organization](/docs/organizations/overview#active-organization) at any given time. Clerk automatically sets organizations as
+active when a user picks an organization with a pre-built component (e.g. the
+[Organization Switcher](/docs/components/organization/organization-switcher)), and Clerk provides
+APIs (like [<code>auth()</code>](/docs/references/nextjs/auth) and [<code>useAuth()</code>](/docs/references/react/use-auth)), that let you determine the active organization and render the appropriate
+content.
```

Comment:
When we reference a component, like `<OrganizationSwitcher />`, the format needs to be as it would be in code.

Also, we prefer to use backticks over `<code>` as its more readable in a MDX file.

 ```suggestion
[`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher)). Additionally, Clerk provides APIs such as [`auth()`](/docs/references/nextjs/auth) and [`useAuth()`](/docs/references/react/use-auth)) that let you determine the active organization and render the appropriate content accordingly.
```